<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWindow</class>
 <widget class="QMainWindow" name="loginWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>142</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>311</width>
    <height>142</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>311</width>
    <height>142</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ورود</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/login.png</normaloff>../icons/login.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mainGroupBox">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>6</y>
      <width>301</width>
      <height>130</height>
     </rect>
    </property>
    <property name="title">
     <string>ورود کاربر</string>
    </property>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>96</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>انصراف</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginPushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>96</y>
       <width>86</width>
       <height>23</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>ورود</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="userLabel">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>22</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>نام کاربری:</string>
     </property>
    </widget>
    <widget class="QLabel" name="passLabel">
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>51</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>رمز عبور:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="passLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="messageLabel">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>74</y>
       <width>207</width>
       <height>16</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
  <tabstop>loginPushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>passLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginPushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
